modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
+- Select would in some situations not have a selected value when expected to. #TINY-9679
```

Comment:
I tried to keep the summary here short, rather than going into a much longer detail. It does become more vague because of it, but I think 'somewhat vague' is better than just really poorly explained to keep it short. It mentions what was changed, and it has a ticket number.